uint32[] id_list
float32[] speed_list
float32 param
uint32 mode
uint32 none